hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -283,10 +281,12 @@ void getAndApplyDeltaUpdatesFromOM(
       LOG.debug("Number of updates received from OM : " +
           dbUpdates.getData().size());
       for (byte[] data : dbUpdates.getData()) {
-        WriteBatch writeBatch = new WriteBatch(data);
-        writeBatch.iterate(omdbUpdatesHandler);
-        RDBBatchOperation rdbBatchOperation = new RDBBatchOperation(writeBatch);
-        rdbBatchOperation.commit(rocksDB, new WriteOptions());
+        try (WriteBatch writeBatch = new WriteBatch(data)) {
+          writeBatch.iterate(omdbUpdatesHandler);
+          RDBBatchOperation rdbBatchOperation =
+              new RDBBatchOperation(writeBatch);
+          rdbBatchOperation.commit(rocksDB, new WriteOptions());
```

Comment:
And this one (`new WriteOptions()`) was not flagged by Sonar.

Reply:
@adoroszlai I will be fixing this in the next sonar patch.